GeoEntity.md

```diff
@@ -0,0 +1,280 @@
+#GeoEntity API
+
+&nbsp;
+
+#Contents
+
+- [Description](#description)
+- [All Countries Request](#all-countries-request)
+- [All Administrative Area Ones Request](#all-administrative-area-ones-request)
+- [All Administrative Area Twos Request](#all-administrative-area-twos-request)
+- [All Localities Request](#all-localities-request)
+- [All Postal Codes Request](#all-postal-codes-request)
+- [All Metropolitan Statistical Areas Request](#all-metropolitan-statistical-areas-request)
+- [Versioning](#versioning)
+
+&nbsp;
+----------------------------
+#Description
+
+This service provides aggregated geography entities. Currently the following requests could be made.
+- Request all countries in the world.
+- Request all administrative area ones in a specific country.
+- In a specific country ( and a specific first order of administrative area as optional), 
+	- Request all administrative area twos.
+	- Request all localities.
+	- Request all postal codes.
+	- Request all metropolitan statistical areas.
+	
+This service supports both HTTP/GET and POST methods.
```

Comment:
no, I mean BS will let you *disable* POSTs. POSTing to this service is especially odd because it's just a static data retrieval service. I would rather it only support GETs.